Doc/library/inspect.rst

```diff
@@ -675,6 +675,12 @@ function.
    Accepts a wide range of Python callables, from plain functions and classes to
    :func:`functools.partial` objects.
 
+   If *follow_wrapped* is ``False`` *callable* will not be unwrapped
+   (``callable.__wrapped__`` will not be used to unwrap decorated callables).
+
+   If *bound_arg* is ``False``, remove ``self`` parameter
+   from the method signature.
```

Comment:
It's better to describe the non-default behaviour for boolean options (so folks know what they'll get if they request non-default behaviour):

```suggestion
   If *bound_arg* is ``True``, report the (already bound) first parameter on bound
   instance  or class methods (usually ``self`` or ``cls``). This emulates the historical
   behaviour of the deprecated :func:`getfullargspec` function.
```

(Describing both behaviours is also an option, but it seemed excessively verbose in this case)